pcsx2-qt/Settings/GameCheatSettingsWidget.cpp

```diff
@@ -167,7 +167,6 @@ void GameCheatSettingsWidget::reloadList()
 		const bool enabled =
 			(std::find(m_enabled_patches.begin(), m_enabled_patches.end(), pi.name) != m_enabled_patches.end());
 
-		const std::string_view name_part = pi.GetNamePart();
```

Comment:
Check with sten, not sure if that was accidental or not.

Reply:
@stenzek you have been summoned.

Reply:
It's fine